lib/crc/crc24_sw.c

```diff
@@ -30,3 +33,62 @@ uint32_t crc24_pgp_update(uint32_t crc, const uint8_t *data, size_t len)
 
 	return crc;
 }
+
+/** This code is originally extracted from https://github.com/go-gnss/rtcm,
```

Comment:
Interestingly this is a violation of coding guideline 24 ("The character sequences /* and // shall not be used within a comment"). That rule seems very impractical though :)